scripts/storybook/src/utils.js

```diff
@@ -170,6 +170,9 @@ function _createCodesandboxRule(allPackageInfo = getAllPackageInfo()) {
 
       return acc;
     }, /** @type import('storybook-addon-export-to-codesandbox').BabelPluginOptions*/ ({}));
+
+    delete importMappings['@fluentui/react-data-grid-react-window'];
```

Comment:
This is kind of a hack, but AFAIK there's no other way to distinguish a converged package that won't be in react-components vs a converged pacakge that will be in react-components

any suggestions @Hotell ?